chapter2/tests/t20/terraform/dev/security_groups.tf

```diff
@@ -0,0 +1,62 @@
+module "wordpress_ec2_sg" {
+  source              = "terraform-aws-modules/security-group/aws"
+  version             = "5.1.0"
+  name                = local.labels.ec2_sg
+  description         = "Security group for the ec2 instances"
+  vpc_id              = data.aws_vpc.target.id
+  ingress_cidr_blocks = [data.aws_vpc.target.cidr_block]
+  ingress_rules       = ["all-all"]
```

Comment:
invalid - you should only allow access by port 80 and only as the source to sg that is alb sg.

Reply:
> invalid - you should only allow access by port 80 and only as the source to sg that is alb sg.

@dmitry-mightydevops
Done
And 22-ssh?

Reply:
where is reply, @rtest12